test/e2e/simple.go

```diff
@@ -32,29 +28,20 @@ func E2ESimple(logger *log.Logger) error {
 	logger.Println("Creating testnet validators")
 	testnet.NoError("failed to create genesis nodes", testNet.CreateGenesisNodes(4, latestVersion, 10000000, 0, testnet.DefaultResources))
 
-	logger.Println("Creating account")
-	kr, err := testNet.CreateAccount("alice", 1e12, "")
-	testnet.NoError("failed to create account", err)
+	logger.Println("Creating txsim")
+	endpoints, err := testNet.RemoteGRPCEndpoints()
+	testnet.NoError("failed to get remote gRPC endpoints", err)
+	err = testNet.CreateTxClient("txsim", testnet.TxsimVersion, 1, "100-2000", 100, testnet.DefaultResources, endpoints[0])
```

Comment:
[Not to be addressed in this PR yet relevant]: `testnet.TxsimVersion` is currently a hardcoded version of txsim. While it works fine for benchmark tests, I believe that for  e2e tests, especially considering our plan to add [this capability](https://github.com/celestiaorg/celestia-app/issues/3509), we need to ensure that we always use the latest version of txsim. We already have an issue for this: https://github.com/celestiaorg/celestia-app/issues/3288.